src/ext/java/org/opentripplanner/ext/vectortiles/layers/vehicleparkings/DigitransitVehicleParkingGroupPropertyMapper.java

```diff
@@ -0,0 +1,51 @@
+package org.opentripplanner.ext.vectortiles.layers.vehicleparkings;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.json.simple.JSONArray;
+import org.json.simple.JSONObject;
+import org.opentripplanner.common.model.T2;
+import org.opentripplanner.ext.vectortiles.PropertyMapper;
+import org.opentripplanner.transit.model.basic.I18NString;
+import org.opentripplanner.transit.model.basic.TranslatedString;
+
+public class DigitransitVehicleParkingGroupPropertyMapper
+  extends PropertyMapper<VehicleParkingAndGroup> {
+
+  public static DigitransitVehicleParkingGroupPropertyMapper create() {
+    return new DigitransitVehicleParkingGroupPropertyMapper();
+  }
+
+  @Override
+  protected Collection<T2<String, Object>> map(VehicleParkingAndGroup parkingAndGroup) {
+    var items = new ArrayList<T2<String, Object>>();
+    var group = parkingAndGroup.vehicleParkingGroup();
+    String parking = JSONArray.toJSONString(
+      parkingAndGroup
+        .vehicleParking()
+        .stream()
+        .map(vehicleParkingPlace -> {
+          JSONObject parkingObject = new JSONObject();
+          parkingObject.put("carPlaces", vehicleParkingPlace.hasCarPlaces());
+          parkingObject.put("bicyclePlaces", vehicleParkingPlace.hasBicyclePlaces());
+          parkingObject.put("id", vehicleParkingPlace.getId().toString());
+          // TODO translate name
+          parkingObject.put("name", vehicleParkingPlace.getName().toString());
+          return parkingObject;
+        })
+        .toList()
+    );
+    // TODO translate name
+    items.addAll(
```

Comment:
Do you need the `items` variable? Could you not simply return the `List.of()`?

Reply:
Thanks for pointing that out, it was left over from when I had to insert the name translations in a separate call. Cleaned it up in https://github.com/opentripplanner/OpenTripPlanner/pull/4510/commits/d601bf84398dd9f704a9c5d588756e6b2092472b